ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-master-presubmits.yaml

```diff
@@ -24,52 +24,23 @@ presubmits:
     - master
     context: conformance
     decorate: true
-    decoration_config:
-      grace_period: 0s
-      skip_cloning: true
-      timeout: 0s
     name: pull-ci-azure-master-e2e-azure-conformance
     rerun_command: /test conformance
     spec:
       containers:
       - args:
-        - --artifact-dir=$(ARTIFACTS)
-        - --give-pr-author-access-to-namespace=true
-        - --secret-dir=/usr/local/e2e-secrets/azure
-        - --target=[output:stable:azure]
-        - --target=cluster-launch-e2e-azure-conformance
-        - --template=/usr/local/cluster-launch-e2e-azure-conformance.yaml
-        command:
-        - ci-operator
+        - hack/tests/origin-conformance.sh
         env:
-        - name: CONFIG_SPEC
-          valueFrom:
-            configMapKeyRef:
-              key: openshift-openshift-azure-master.yaml
-              name: ci-operator-master-configs
-        - name: TEST_COMMAND
-          value: TEST_SUITE=openshift/conformance/parallel run-tests
-        - name: TEST_IMAGE
-          value: registry.svc.ci.openshift.org/openshift/origin-v3.11:tests
-        image: ci-operator:latest
+        - name: TEST_NODES
+          value: "30"
```

Comment:
I'm not familiar with the OpenShift conformance tests. Does this mean it's going to spin up 30 test nodes? Do we have enough quota for that?

Reply:
No, I think that's how many ginkgo threads it starts, I took it straight from the current value in the launch template https://github.com/openshift/release/blob/12832438b0b90ede1d48a9632fd9e33610e58f9b/ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure.yaml#L200

Reply:
Ah, okay. We're only giving the container 10m CPU so that might be too much parallelism, but I guess we'll find out if it fails.